src/components/LikeThought.jsx

```diff
@@ -0,0 +1,49 @@
+import { useEffect, useState } from 'react'
+import PropTypes from 'prop-types'
+import './Thoughts.css'
+import { GetThoughts } from './Thoughts'
+
+export const LikeThoughts = ({ index }) => {
+	const [likes, setLikes] = useState(index.hearts)
```

Comment:
I think your tab indentation setting in VS code is now four-space. Think better to change two-space instead to align the standard to avoid unnecessary changes when you work with someone :)